packages/grub-config/config/bootargs.cfg

```diff
@@ -1,3 +1,8 @@
 set kernel=/boot/vmlinuz
-set kernelcmd="console=tty1 root=LABEL=$label iso-scan/filename=$img panic=5 security=selinux selinux=1"
+if [ -n "$recoverylabel" ]; then
+    set kernelcmd="console=tty1 root=live:CDLABEL=$recoverylabel rd.live.dir=/ rd.live.squashimg=$img panic=5"
```

Comment:
I'd say this doesn't boot for two reasons:

* According to the installer I'd say the `rd.live.dir` should be set to `rd.live.dir=/cOS`
* I'd say there is no such a CDLABEL` for the `COS_RECOVERY filesystem, since it is not an ISO. I'd use LABEL instead. Probaby dracut code uses CDLABEL and LABEL as the same thing though...

In case it fails to boot I's suggest to use something like:

```
set kernelcmd="console=tty1 root=live:LABEL=$recoverylabel rd.live.dir=/cOS rd.live.squashimg=$img panic=5"
```

Reply:
it does boot, there are tests for it :)

Reply:
I didn't touched here where we do put the squashfs files, so not sure if touching `rd.live.dir` makes sense.. or I'm missing something? I think it's just working because `rd.live.squashimg` contains the full path (`/cOS/...`)

Reply:
changed from CDLABEL to LABEL done in https://github.com/rancher-sandbox/cOS-toolkit/pull/241/commits/70d4638cb5179d37b2dbc754ef1d83f16fae2149 :+1: